Add section on how to reset desktop wallet data (#508)
Concordium/concordium.github.io: source/mainnet/net/desktop-wallet/reset-data.rst

.. _reset-data:

==============================
Reset desktop wallet data
==============================

> All the local data of the desktop wallet is stored in a database in the applications user data folder.

Reviewer: ```suggestion
All the local data of the desktop wallet is stored in a database in the user's applications data folder.
```

> The desktop wallet does not remove your user data when it is uninstalled, to avoid users losing their data, and to simplify the update process of the application.

Reviewer: ```suggestion
The desktop wallet does not remove your user data when it is uninstalled; this avoids users losing their data, and simplifies the update process for the application.
```

> We will here describe the process of deleting this user data, because it might be necessary to do, but you should make sure that you have a :ref:`backup <export-import>` with your data, before doing so.

Reviewer: ```suggestion
Occasionally, it might be necessary to delete the user data. The following steps describe the process of deleting this user data. Make sure that you have a :ref:`backup <export-import>` with your data before starting this process.
```

> .. warning:: Deleting the user data without a :ref:`backup <export-import>` will mean you can only recover what is recoverable with :ref:`account recovery <export-import>` (under "Recover accounts without a backup file"), which does not include your identities, address book, names or notes.

Reviewer: I think you can make a link to this. I have done that in the Use the Desktop Wallet topic here: https://developer.concordium.software/en/mainnet/net/desktop-wallet/accounts-desktop.html#recover-existing-accounts. You can see it links back to the export import topic with account recovery highlighted.

Author: Hmm, I checked the repo of sphinx-tabs, and it didn't seem that they supported it yet.

When I try the link it looks weird, it just highlights every occurrence of the words account and recovery, though?
https://developer.concordium.software/en/mainnet/net/guides/export-import.html?highlight=account%20recovery

Author: ![Screenshot from 2022-06-07 15-37-23](https://user-images.githubusercontent.com/87635671/172394243-d5adc46a-1d6a-401b-9dfc-b1ce9c06c5f7.png)

Reviewer: Yes, I see that now.

> Deleting the user data

Reviewer: ```suggestion
Delete the user data
```
